### ISSUE TITLE
Turbosnap not trigger on dependency update

### ISSUE THREAD

**Issue by Ankoge** [chromaui/chromatic-cli#927]
We had a dependency update ("@chakra-ui/storybook-addon": "5.1.0") and that update cause an error in all storybook component, but no turbosnap made so the PR got an approve.

**Comment by cgbl-90** [comment]
@Ankoge could you reach out to our support chat so we can dig a little deeper?

**Comment by mmuth** [comment]
HI @Ankoge 

I am currently struggling with similar issues (and already contacted the support)... 

My observation is, that I have troubles with multiple package.json files in the same repo. I can configure base-urls/config-urls etc. whatever I want, but only changes to the package.json in the root directory of the repo trigger dependency-changes... Unfortunately all my code/storybook/chromatic is some levels deeper in the file tree. 

I can update you if you have more info. @cgbl-90 if you have further ideas, I'd be glad (well, taking a look at your name I assume I am writing with you over the support :D)

**Comment by mmuth** [comment]
@Ankoge i hope i don't hijack your thread, but just adding some details on the things i discovered (possibly they fit to your troubles)... 

I enabled the `--debug` flag and changed something in the package.json.. Then the lock looks like: 
```
Traversing dependencies for 2 files that changed since the last build
21:33:06.435  {"packageMetadataChanges":[{"changedFiles":["service/microfrontend/package.json"],"commit":"16013627f01a5ed76098fdab88f98778be28c34d"}]} Finding changed dependencies for 1 baselines
21:33:06.443  {"rootPath":"/home/my-git-root/git/my-repo","rootManifestPath":"package.json","rootLockfilePath":"yarn.lock"} Found manifest and lockfile
21:33:06.446  {"pathPairs":[["package.json","yarn.lock"]]} Found 1 manifest/lockfile pairs to check
21:33:06.446  {"filteredPathPairs":[]} Found 0 manifest/lockfile pairs to diff
```

And then it decided that no dependencies were changed (which is, in fact, wrong)...

**Comment by JonathanKolnik** [comment]
Appreciate it @mmuth! Hopefully @cgbl-90 and the team will get you sorted out.

@Ankoge if you could also submit the results of the publish with --debug set that would be super helpful in debugging this. We can probably assist you better via support chat.

**Comment by winkerVSbecks** [comment]
Closing since this was moved to support

**Comment by tevanoff** [comment]
@Ankoge @mmuth Just wanted to let you know that a fix for this went out in v11.3.4. Thanks for reporting!
